pdns/dnsproxy.cc

```diff
@@ -260,7 +260,7 @@ void DNSProxy::mainloop()
 
         DNSPacket p(false),q(false);
         p.parse(buffer,(size_t)len);
-        q.parse(buffer,(size_t)len);
+        //q.parse(buffer,(size_t)len); // removed.  never used?
```

Comment:
@Habbie is this ok to remove?  It didn't seem to be used anywhere.

Reply:
yes